.travis.yml

```diff
@@ -18,6 +18,14 @@ matrix:
     install: make bootstrap
     script:
     - make dependencies test
+  - language: ruby
+    rvm: 2.3.4
+    cache: bundler
+    services: docker
+    bundler_args: --without development
+    env: PUPPET_GEM_VERSION="~> 4.0"
+    install: cd contrib/puppetlabs-lumogon && make bootstrap
+    script: make dependencies test
```

Comment:
This is a style pref, for sure. I like the explicit even though it exists in the Makefile. Just so that I know what's happening here as part of due course before I even have to dig in when I'm troubleshooting. It's benigin, also matches what is already on L13 (same situation as well with deps)